frontend/src/App.tsx

```diff
@@ -41,14 +59,41 @@ function App() {
 
     useEffect(() => {
         getAllWorkouts()
+        getUser()
     }, [])
 
+    function login(provider: "google" | "github") {
+        const host = window.location.host === 'localhost:5173' ? 'http://localhost:8080' : window.location.origin
+
+        window.open(host + '/oauth2/authorization/' + provider, '_self')
+
+        setLoggedIn(true);
+    }
+
+    function logout() {
+        const host = window.location.host === 'localhost:5173' ? 'http://localhost:8080' : window.location.origin
+
+        window.open(host + '/logout', '_self')
+    }
+
     return (
         <>
             <div className="NAVBAR">
 
                 <Link to="/"><h1>WORKOUT BUDDY</h1></Link>
+
                 <Link to="/add">Add Workout</Link>
+                {(!loggedInUser || loggedInUser === 'anonymousUser') &&
+                    <button onClick={() => login("google")}>Google Login</button>}
+                {(!loggedInUser || loggedInUser === 'anonymousUser') &&
+                    <button onClick={() => login("github")}>Github Login</button>}
```

Comment:
How can we Put this under one Button what pops up and you can choose the Provider?
I would like to make it a little more Pretty